lib/engines/content_block_manager/app/controllers/content_block_manager/content_block/documents/schedule_controller.rb

```diff
@@ -8,12 +8,14 @@ def edit
 
   def update
     document = ContentBlockManager::ContentBlock::Document.find(params[:document_id])
-    @content_block_edition = document.latest_edition
-    validate_scheduled_edition
+    previous_edition = document.latest_edition
```

Comment:
this is definitely an improvement! only thing is when we duplicate, the state is in `published` and that would stop the cancel/delete jobs working if the user abandoned the form. think it can be solved by setting state as `draft` (also think there must be some test safety nets missing as this wasn't caught, only saw it as I tried it out in the UI)

Reply:
I would also double check that changing the state on this new edition doesn't create a new `version` and thus an entry on the timeline 🤔 maybe there's a clever rails way of duplicating except for one field?

Reply:
I think that's fine, as it shows as it's superseded on the timeline, which I think is the correct behaviour. We can revisit if it's confusing